config/routes.rb

```diff
@@ -1,6 +1,7 @@
 require 'api_constraints'
 
 Foreman::Application.routes.draw do
+  mount JasmineRails::Engine => '/specs' if defined?(JasmineRails)
```

Comment:
Maybe, again this is the default so that you can just run 'rails s' then go to /specs and check tests. I don't mind changing it